src/renderer/services/dex.js

```diff
@@ -1274,38 +1279,67 @@ export default {
         const dexAddress = wallet.getAddressFromScriptHash(assets.DEX_SCRIPT_HASH);
         neo.fetchSystemAssetBalance(dexAddress, null, false)
           .then(async (balance) => {
+            // console.log(`calculateWithdrawInputsAndOutputs fetched system asset balance to calculate withdraw inputs / outputs. ${JSON.stringify(balance)}`);
             config.balance = balance;
             const unspents = assetId === assets.GAS ? config.balance.assets.GAS.unspent : config.balance.assets.NEO.unspent;
             await this.checkUnspentsReservedState(assetId, unspents);
 
-            let inputTotal = new BigNumber(0);
-            _.sortBy(unspents, [unspent => Math.abs(unspent.value.minus(quantity).toNumber())]).forEach((currentUnspent) => {
-              if (inputTotal.isGreaterThanOrEqualTo(quantity)) {
-                return;
-              }
+            const pickedInputs = [];
+            const pickedUnspents = [];
+            let quantitySumOfPickedInputs = new BigNumber(0);
+            _.orderBy(unspents, [unspent => parseFloat(unspent.value.toString())], ['desc']).some((currentUnspent) => {
```

Comment:
For some reason just using unspent.value does not work -- it sorts it wrong -- this seems to work reliably